src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java

```diff
@@ -207,6 +207,18 @@ private static FeedbackQuestionAttributes extractFeedbackQuestionData(Map<String
             newQuestion.numberOfEntitiesToGiveFeedbackTo = Const.MAX_POSSIBLE_RECIPIENTS;
         }
         
+        if (newQuestion.questionType != FeedbackQuestionType.CONTRIB
```

Comment:
We need to remove the 'custom' feedback path option on the UI for contribution questions then